### ISSUE TITLE
bottle BaseRequest.MAX_PARAMS causes HTTPError: 400 Client Error: Bad Request on packages with more than 100 requirements.

### ISSUE THREAD

**Issue by bartekbrak** [pypiserver/pypiserver#82]
Hi. 
Geez, this was a nightmare to track down ;). 
Look at [BaseRequest.MAX_PARAMS](https://github.com/pypiserver/pypiserver/blob/master/pypiserver/bottle.py#L900)

We have a package with 112 requirements at the company, this caused [HTTPError: 400 Client Error: Bad Request](https://github.com/pypiserver/pypiserver/blob/master/pypiserver/_app.py#L252) as this exceeded allowed POST parameters.

You can reproduce by building this fake package and uploading it (I used twine):

```
from setuptools import setup
setup(
    name='pyserver_test',
    install_requires=['a==1.0'] * 200,
)
```

```
$ pip wheel .
$ twine upload -r my_locally_run_pypiserver wheelhouse/pyserver_test-0.0.0-cp27-none-linux_x86_64.whl 
Uploading distributions to http://127.0.0.1:8080
Uploading pyserver_test-0.0.0-cp27-none-linux_x86_64.whl
HTTPError: 400 Client Error: Bad Request
```

Please explain why cap at 100 (I assume this is a question to bottle authors). I can easily fix this in my fork but I don't have an idea how to elegantly fix it for everyone else.

**Comment by ankostis** [comment]
Assuming it is linked to this one bottlepy/bottle#366.

**Comment by ankostis** [comment]
The MAX_PARAMS limit has been fixed to bottle >= 0.12 (`pypiserver` uses 0.11.6).
We need to update `bottle`.
